pinot-broker/src/main/java/com/linkedin/pinot/broker/queryquota/TableQueryQuotaManager.java

```diff
@@ -234,7 +239,58 @@ public void cleanUpRateLimiterMap() {
     _rateLimiterMap.clear();
   }
 
+  /**
+   * Process query quota change when number of online brokers has changed.
+   */
   public void processQueryQuotaChange() {
-    // TODO: update rate
+    long startTime = System.currentTimeMillis();
+
+    ExternalView currentBrokerResource = HelixHelper.getExternalViewForResource(_helixManager.getClusterManagmentTool(),
+        _helixManager.getClusterName(), BROKER_RESOURCE_INSTANCE);
```

Comment:
This can be `null` in certain cases, should log a warning and exit cleanly here.